### PR TITLE
Add a patch/unpatch example for the Python API

### PR DESCRIPTION
**Pull request by simark**
It serves as an example and it's useful as a quick way to test is things
are working.

### CODE REVIEW

File: python/examples/ola_patch_unpatch.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Library General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ola_patch_unpatch.py
+# Copyright (C) 2005 Simon Newton
```
Comment (peternewman) [inline python/examples/ola_patch_unpatch.py]: Just you again.

File: python/examples/ola_patch_unpatch.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+"""Patch and unpatch ports."""
+
+from __future__ import print_function
+from ola.ClientWrapper import ClientWrapper
+from ola.OlaClient import OlaClient
+import argparse
+
+__author__ = 'nomis52@gmail.com (Simon Newton), ' \
```
Comment (peternewman) [inline python/examples/ola_patch_unpatch.py]: Just you.

File: python/examples/ola_patch_unpatch.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+             'simon.marchi@polymtl.ca (Simon Marchi)'
+
+
+def ParseArgs():
+    description='Example program that shows how to patch or unpatch a port.'
```
Comment (peternewman) [inline python/examples/ola_patch_unpatch.py]: I'd drop the example bit, its a fully functioning app isn't it?

**Comment by peternewman** [comment]
LGTM, @nomis52 

One thing @simark , I'd rather see the commit history please, as opposed to them being squashed, otherwise I have to review the entire thing each time. If you don't squash them, I can just review the changes to fix comments/other issues.

**Comment by simark** [comment]
Ok, so next time I should add commits on top of the initial one when addressing comments?

File: python/examples/ola_patch_unpatch.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                           choices=['input', 'output'],
+                           required=True)
+    argparser.add_argument('--action', '-a',
+                           choices=['patch', 'unpatch'],
+                           required=True)
+
+    return argparser.parse_args()
+
+
+def PathPortCallback(status):
```
Comment (nomis52) [inline python/examples/ola_patch_unpatch.py]: PatchPortCallback , PathPort is something different :)

**Comment by nomis52** [comment]
Just the one comment

File: python/examples/ola_patch_unpatch.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    if status.Succeeded():
+        print('Success!')
```
Comment (nomis52) [inline python/examples/ola_patch_unpatch.py]: 2 character indents please

**Comment by peternewman** [comment]
Yes please @simark , just make another commit with the fixes from the comments, and push that, so we then have two commits, we can see the overall change via GitHub, and the changes to address our comments only by viewing the second commit.

**Comment by simark** [comment]
Done!

**Comment by nomis52** [comment]
One thing I forgot, you need to add these to the Makefile.mk in the same directory - otherwise they won't be included in the tarball.

**Comment by nomis52** [comment]
Other than that this looks fine.

**Comment by simark** [comment]
Ah, of course. Done.

**Comment by nomis52** [comment]
Thanks!

**Pull request merged**